<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRecordWidget</class>
 <widget class="QWidget" name="NewRecordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Новый рецепт</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../src/icon.png</normaloff>../src/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditRecordWidget {background-color: rgb(252, 238, 251);}
QPushButton {background-color: rgb(239, 187, 238);
	border-radius: 10px;
	border-width: 0px;}
QPushButton:hover {background-color: rgb(244, 204, 242);
	border-width: 0px;}
QLineEdit {background-color: rgb(239, 187, 238);}
#nameLine {border-radius: 10px;
	border-width: 1px;
	min-height: 40px;
	max-height: 40px;
	min-width: 200px;
	max-width: 200px}

QTextEdit  {background-color: rgb(255, 255, 255);}
</string>
  </property>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addPicButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Загрузить изображение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectCategoryButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Категория</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="recipeBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>590</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить рецепт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>590</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>303</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="ingredientsBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>14</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewRecordWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
